src/Components/ImageChoice/ImageChoice.js

```diff
@@ -0,0 +1,94 @@
+import React, {useRef} from 'react';
+import './ImageChoice.scss';
+
+const ImageChoice=({ qno, data, id, isEdit = false, onUpdate})=> { 
+
+    const onFieldUpdate = (field,value,index) => {
+        let tempData = { ...data };
+        if(field==="TITLE"){
+            tempData.title = value; 
+        }
+        if(field==="SELECTED"){
+            tempData.tabs[index].selected = !tempData.tabs[index].selected;
+        }
```

Comment:
I put a comment on other PR for this, just handle selected one as above and other value update as mentioned in other PR. tempData[field] = value --> this needs to be inside condition